sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/AnalyzedDocument.java

```diff
@@ -11,7 +11,7 @@
 /**
  * An object describing the location and semantic content of a document.
  */
-public final class AnalyzedDocument {
+public class AnalyzedDocument {
```

Comment:
no longer final class?

Reply:
yeah, might need it for a future design to be extendable. Will resolve this before GA.